src/pages/AdminDashboard/roomsActions.ts

```diff
@@ -0,0 +1,65 @@
+import { RoomsActionsTypes, SET_ROOMS } from './roomsActionTypes'
+import { Api } from '../../services/api'
+import { showErrorMessage } from '../../components/App/Snackbar/snackbarActions'
+import {
+    hideBackdrop,
+    showBackdrop,
+} from '../../components/App/Backdrop/backdropActions'
+
+export const setRooms = (rooms: any): RoomsActionsTypes => ({
+    type: SET_ROOMS,
+    payload: {
+        rooms,
+    },
+})
+
+export const getRooms = (t) => async (dispatch, getState) => {
+    const { user: userState } = getState()
+    const { token } = userState.user
```

Comment:
getTokenSelector isn't available there?

Reply:
if you mean `useSelector(selectToken)`, nope it's not available in the async action ^^